packages/discovery-provider/integration_tests/tasks/user_bank_mock_transactions.py

```diff
@@ -412,6 +419,178 @@
     )
 )
 
+# Purchase of track id 1 for $1 USDC with purchaser user id 2 including download access
+ock_valid_track_purchase_tx = GetTransactionResp.from_json(
+    json.dumps(
+        {
+            "jsonrpc": "2.0",
+            "result": {
+                "slot": 227246439,
+                "transaction": {
+                    "signatures": [MOCK_SIGNATURE],
+                    "message": {
+                        "header": {
+                            "numRequiredSignatures": 1,
+                            "numReadonlySignedAccounts": 0,
+                            "numReadonlyUnsignedAccounts": 8,
+                        },
+                        "accountKeys": [
+                            FEE_PAYER,
+                            SENDER_USDC_USER_BANK_ADDRESS,
+                            NONCE_ACCOUNT_ADDRESS,
+                            RECIPIENT_USDC_USER_BANK_ADDRESS,
+                            "11111111111111111111111111111111",
+                            CLAIMABLE_TOKENS_PDA,
+                            USDC_PDA,
+                            "KeccakSecp256k11111111111111111111111111111",
+                            "Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo",
+                            "Sysvar1nstructions1111111111111111111111111",
+                            "SysvarRent111111111111111111111111111111111",
+                            "TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA",
+                        ],
+                        "recentBlockhash": "5H434VMiHgK7RaJZaBKKcriu4eky8erb9QGfcHJSZquU",
+                        "instructions": [
+                            {
+                                "programIdIndex": 7,
+                                "accounts": [],
+                                "data": "H4eCheRWTZDTCFYUcyMzE6EhQMZvvvLKJ9g6YaUpbZeoLLgVj1uvwCTdzcb2MzbKHsRjN8DjLYdqxuQEZe2TjUKCuBMrFtpnnLd4RcvBnr4ieHCdH8ZU1N6XDfiqyKB4zenQ9S4viza4ob4gbtmiRS6o6KGEtL3fJQRvaA3tdtSx1rfFogZzwMXAxHrkuxHrpAqfm",
+                                "stackHeight": None,
+                            },
+                            {
+                                "programIdIndex": 5,
+                                "accounts": [0, 1, 3, 2, 6, 10, 9, 4, 11],
+                                "data": "6dMrrkPeSzw2r5huQ6RToaJCaVuu",
+                                "stackHeight": None,
+                            },
+                            {
+                                "programIdIndex": 8,
+                                "accounts": [0],
+                                "data": PURCHASE_TRACK1_MEMO_DATA,
```

Comment:
The comment above this entry says 'including download access' but this is using the legacy memo with no access specified?

Reply:
this whole thing was bunk, i accidentally duplicated the first mock tx.